packages/editor/src/utils/url.js

```diff
@@ -45,7 +45,7 @@ export function cleanForSlug( string ) {
 	return trim(
 		deburr( string )
 			.replace( /[\s\./]+/g, '-' )
-			.replace( /[^\w-]+/g, '' )
+			.replace( /[^\p{L}\p{N}_-]+/gu, '' )
```

Comment:
The docs say:
```
 * Converts Latin-1 Supplement and Latin Extended-A letters to basic Latin * letters. Removes combining diacritical marks. Converts whitespace, periods, * and forward slashes to hyphens. Removes any remaining non-word characters * except hyphens. Converts remaining string to lowercase. It does not account * for octets, HTML entities, or other encoded characters.

*
```

Should we update the docs?

Reply:
Thank you for the review @jorgefilipecosta!

Updated the docs:
```
 * Converts Latin-1 Supplement and Latin Extended-A letters to basic Latin
 * letters. Removes combining diacritical marks. Converts whitespace, periods,
 * and forward slashes to hyphens. Removes any remaining non-word characters
 * except hyphens and underscores. Converts remaining string to lowercase.
 * It does not account for octets, HTML entities, or other encoded characters.
```

The addition is "except hyphens **and underscores**". I believe the rest of it is accurate :+1: